genfragments/PbPb_5p36TeV/Starlight/STARLIGHT_QED_tau-fragment.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+    args = cms.vstring('/cvmfs/cms.cern.ch/phys_generator/gridpacks/RunIII/5p36TeV/starlight/starlight_ditau_el8_amd64_gcc11_CMSSW_13_0_18_HeavyIon_tarball.tgz'),
+    nEvents = cms.untracked.uint32(1),
+    numberOfParameters = cms.uint32(1),
+    outputFile = cms.string('cmsgrid_final.lhe'),
+    scriptName = cms.FileInPath('GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh')
+)
+
+#Link to datacards:
+#https://github.com/cms-sw/genproductions/blob/master/bin/Starlight/production/PbPb_5p36TeV/starlight_ditau.in
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+from GeneratorInterface.ExternalDecays.TauolaSettings_cff import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+    ExternalDecays = cms.PSet(
+        Tauola = cms.untracked.PSet(
+            TauolaPolar,
+            TauolaDefaultInputCards
+        ),
+        parameterSets = cms.vstring('Tauola')
+    ),
+    UseExternalGenerators = cms.untracked.bool(True),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        skip_hadronization = cms.vstring(
+            'ProcessLevel:all = off',
+            'Check:event = off'
+        ),
+        parameterSets = cms.vstring('pythia8CommonSettings','pythia8PSweightsSettings','skip_hadronization')
+    ),
+    comEnergy = cms.double(5362.0),
+    filterEfficiency = cms.untracked.double(1.0),
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    pythiaPylistVerbosity = cms.untracked.int32(1)
+)
+
+ProductionFilterSequence = cms.Sequence(generator)
```

Comment:
why tau process doesn't need the filter like electron and muon channels

Reply:
I could add a filter. Just wasn't sure how to handle since the tau is not a final product

Reply:
Interesting point and I might have some hint...

https://github.com/cms-sw/cmssw/blob/6e82cc8aa6fff019b94eb34a5b5297c2eaa62573/GeneratorInterface/Core/interface/EmbeddingHepMCFilter.h#L10 this is the so-called embeddinghepmcfilter that should be used for tau embedding samples? but i dont know if it plays well with tauola

Reply:
Here exo has an example usage: https://gitlab.cern.ch/cms-exo-mci/EXO-MCsampleRequests/-/blob/master/genFragments/Hadronizer/13p6TeV/GluGluPhiTauMuTauH/GluGluPhiTauMuTauH_M100_Fragment.py?ref_type=heads

Reply:
Just checked and this doesn't seem to work.. since `tauola` runs the filter being applied... Need another handle then...